api/src/filter.js

```diff
@@ -7,6 +7,10 @@ export function transformStringFilter (op) {
     return { $regex: op.contains }
   }
 
+  if (op.in) {
```

Comment:
This method seems more like a common filter than a string filter, so just putting it in here.